dlrover/python/common/multi_process.py

```diff
@@ -471,28 +497,38 @@ def __init__(self, name="", create=False):
             name=f"shard_dict_{name}", create=self._create
         )
 
+    def deal_client_msg(self, connection):
```

Comment:
There are 3 methods with the same name `deal_client_msg`. We can rename them as
`_deal_shared_lock_msg`
`_deal_shared_queue_msg`
`_deal_shared_dict_msg`

Reply:
ok